Importer/Interpreter/EntityDataInterpreter.php

```diff
@@ -273,6 +275,21 @@ protected function getEntity(array $interpretedData)
     }
 
     /**
+     * @param object $entity
+     *
+     * @return ConstraintViolationListInterface
+     */
+    private function validate($entity)
+    {
+        $violationList = $this->validator->validate($entity);
+
+        $event = new PostImportValidationEvent($violationList, $entity);
+        $this->eventDispatcher->dispatch(ImportEvents::POST_VALIDATION, $event);
```

Comment:
yeha like that approach. Than it is on validation. Modify is a wide term, lets they they can add new ones. I will have a though.